.github/workflows/python.yaml

```diff
@@ -68,7 +66,11 @@ jobs:
       run: |
         brew install ccache curl
       # TODO: Does brew install need bzip2, libz-dev, and make?
-    
+
```

Comment:
The TODO above seems orphaned.

Reply:
I'm gonna do some CI clean up once I have it working. I'll get this then.